deploy.py

```diff
@@ -182,6 +174,28 @@ def notarizefile(filetonotarize, username, password):
     return currentstatus
 
 
+def find_qt():
+    """ Common code between Linux and Darwin """
+    with msg_printer('Checking qtdir...'):
```

Comment:
So this is a "context manager" that if you don't throw a `MsgPrinterException` then you will get the happy checkmark when you come out of the context, automatically.